.github/workflows/docker.yml

```diff
@@ -0,0 +1,54 @@
+name: Push Docker images to DockerHub and ECR
```

Comment:
Could you rename this to `deploy.yml`? (GH doesn't me do it as suggestion). `ci.yml` also uses docker so it's clearer if this one has a different name, imo.

Reply:
Done!